recipes/beyla/README.md

```diff
@@ -0,0 +1,141 @@
+# eBPF HTTP Observability with Beyla
+
+This recipe demonstrates how to configure the OpenTelemetry Collector
+(as deployed by the Operator) to send [Beyla](https://github.com/grafana/beyla)
+metric and trace data to GCP [Cloud Trace](https://cloud.google.com/trace) and
+[Google Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus).
+
+In this recipe, Beyla is configured to collect http metrics and traces from all
+workloads in the cluster without any code changes. Beyla has other features,
+such as auto-instrumentation for Go applications, but this sample does not use
+it for that purpose.
+
+This recipe is based on applying a Collector config that enables the Google Cloud exporter.
+It provides an `OpenTelemetryCollector` object that, when created, instructs the Operator to
+create a new instance of the Collector with that config. If overwriting an existing `OpenTelemetryCollector`
+object (i.e., you already have a running Collector through the Operator such as the one from the
+[main README](../../README.md#starting-the-collector)), the Operator will update that existing
+Collector with the new config.
+
+
+## Prerequisites
+
+* Cloud Trace and Cloud Monitoring APIs enabled in your GCP project
+* The `roles/cloudtrace.agent` and `roles/monitoring.metricWriter`
+  [IAM permissions](https://cloud.google.com/trace/docs/iam#roles)
+  for your cluster's service account (or Workload Identity setup as shown below).
+* A running (non-autopilot) GKE cluster
+* The OpenTelemetry Operator installed in your cluster
+* A Collector deployed with the Operator (recommended) or a ServiceAccount that can be used by the new Collector.
+  * Note: This recipe assumes that you already have a Collector ServiceAccount named `otel-collector`,
+    which is created by the operator when deploying an `OpenTelemetryCollector` object such as the
+    one [in this repo](../../collector-config.yaml).
+* An application already deployed that makes or serves HTTP requests:
+  * Note: Beyla does not currently support HTTPs or gRPC
+  * [One of the sample apps](../../sample-apps) from this repo, without auto-instrumentation enabled.
+
+Note that the `OpenTelemetryCollector` object needs to be in the same namespace as your sample
+app, or the Collector endpoint needs to be updated to point to the correct service address.
+
+## Running
+
+### Workload Identity Setup
+
+If you have Workload Identity enabled (on by default in GKE Autopilot), you'll need to set
```

Comment:
Why do we mention Autopilot here, after saying in the intro that we don't support Autopilot?